crmsh/utils.py

```diff
@@ -2999,9 +3162,9 @@ def get_property(name, property_type="crm_config"):
     """
     if property_type == "crm_config":
         cib_path = os.getenv('CIB_file', constants.CIB_RAW_FILE)
-        cmd = "CIB_file={} crm configure get_property {}".format(cib_path, name)
+        cmd = "CIB_file={} sudo crm configure get_property {}".format(cib_path, name)
```

Comment:
`sudo` preserve only white-listed environmental variables.

```suggestion
        cmd = "CIB_file={} sudo -E CIB_file crm configure get_property {}".format(cib_path, name)
```